src/Sorter/UnresolvedFixtureDependency.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\Sorter;
+
+class UnresolvedFixtureDependency extends \OutOfRangeException
+{
+    public function __construct(string $name = '', int $code = 0, ?\Throwable $previous = null)
```

Comment:
I don't think the name should be optional:

```suggestion
    public function __construct(string $name, int $code = 0, ?\Throwable $previous = null)
```

Reply:
Resolved in #14